### PR TITLE
Adding to dynamodb and showing as a table

### LINKED ISSUE #16: As a user, I want to view a complete list of all public services with basic information about each entry

### PR DESCRIPTION
**Pull request by shub-garg**
image

### CODE REVIEW

**Review by omerfarukbasar — APPROVED** [review]
Table display seems to be working as intended.

File: Database/db-fetch.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import json
+from decimal import Decimal
+
+class DecimalEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, Decimal):
+            return float(obj)  # Convert Decimal to float for JSON encoding
+        return super(DecimalEncoder, self).default(obj)
+
+dynamodb = boto3.resource('dynamodb', region_name='us-east-1')  # Replace with your region
```
Comment (KartikeySharma) [inline Database/db-fetch.py:11]: Do we need to keep comments if the regions have been replaced as per our project needs? (just confirming, not a blocker)

Comment (KartikeySharma) [inline Database/db-fetch.py:11]: Do we need to keep comments if the regions have been replaced as per our project needs? (just confirming, not a blocker)

File: Database/db-foods.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+# Custom JSON encoder for Decimal objects
+class DecimalEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, Decimal):
+            return str(obj)  # Convert Decimal to string for JSON encoding
+        return super(DecimalEncoder, self).default(obj)
+
+# Initialize DynamoDB resource
+dynamodb = boto3.resource('dynamodb', region_name='us-east-1')  # Replace with your actual region
```
Comment (KartikeySharma) [inline Database/db-foods.py:15]: same for all comments asking to replace with our region and table name

**Review by KartikeySharma — APPROVED** [review]

File: Database/db-fetch.py (hunk 1 above)
Comment (shub-garg) [reply Database/db-fetch.py:11]: Ill remove them

Comment (KartikeySharma) [reply Database/db-fetch.py:11]: Thanks, will squash and merge. Everything else LGTM!

**Comment by KartikeySharma** [comment]
PR merged, will address the comments issue in a later fix

**Comment by ahhcash** [comment]
for book keeping, part of #26

**Pull request merged**